web/ui/templates/targets.html

```diff
@@ -44,10 +44,10 @@ <h2 id="targets">Targets</h2>
                         </span>
                       </td>
                       <td>
-                        <span class="cursor-pointer" data-toggle="tooltip" title="" data-html=true data-original-title="<b>Before relabeling:</b>{{range $k, $v := .DiscoveredLabels}}<br>{{$k | html | html}}=&quot;{{$v | html | html}}&quot;{{end}}">
+                        <span class="cursor-pointer" data-toggle="tooltip" title="" data-html=true data-original-title="<b>Before relabeling:</b>{{range $k, $v := .DiscoveredLabels}}<br>{{$k}}=&quot;{{$v}}&quot;{{end}}">
```

Comment:
For some reason, pulling the HTML-escaping into a separate variable seems to work:

    {{$ev := $v | html}}&quot;{{$ev}}&quot;

...I have no idea why, but would like to know :)